pages/api/v2/bots/[id]/index.ts

```diff
@@ -76,6 +76,12 @@ const Bots = RequestHandler()
 				message: '디스코드 서버에 참가해주세요.',
 				errors: ['봇 신청하시기 위해서는 공식 디스코드 서버에 참가해주셔야합니다.'],
 			})
+		else if (result === 5)
+			return ResponseWrapper(res, {
+				code: 403,
+				message: '더 이상 해당 봇에 대한 심사 요청을 하실 수 없습니다.',
+				errors: ['해당 봇은 심사에서 3번 거부되었습니다. 더 이상의 심사를 요청하실 수 없습니다.'],
```

Comment:
워딩 수정 및 이의제기 권리의 안내
```suggestion
				errors: ['해당 봇은 심사에서 3회 이상 거부되었습니다. 더 이상의 심사를 요청하실 수 없습니다.', '이의 제기의 경우 문의해주세요.'],
```
이건 선택적으로 수용하시면 될 것 같아요.

Reply:
이걸로 바꾸는게 더 자연스러울 것 같네요.